conans/client/tools/scm.py

```diff
@@ -203,7 +213,18 @@ def excluded_files(self):
         return excluded_list
 
     def get_remote_url(self):
-        return self.run("info --show-item url").strip()
+        url = self.run("info --show-item url").strip()
+        revision = self.run("info --show-item revision").strip()
+        return "{url}@{revision}".format(url=url, revision=revision)
```

Comment:
We agreed that we should return just the _url_ in `(SVN|Git).get_remote_url`, and in function `SVN.get_qualified_remote_url` return the `url+rev` (like it is). So here, in `get_remote_url`, we must remove the `revision` part.

Reply:
Sorry, forget to put it back to the old state